src/main/java/net/pms/formats/v2/AudioProperties.java

```diff
@@ -219,4 +219,15 @@ public static int getSampleFrequencyFromLibMediaInfo(String mediaInfoValue) {
 			return result;
 		}
 	}
+
+	@Override
+	public String toString() {
+		StringBuilder result = new StringBuilder();
+		result.append("Channel(s): ").append(getNumberOfChannels());
+		if (getAudioDelay() != 0) {
+			result.append(", Delay: ").append(getAudioDelay());
+		}
+
```

Comment:
@Sami32 You removed ```result.append(", Sample Frequency: ").append(getSampleFrequency()).append(" Hz");```` - I disagree. Remember that even though you already have the sample frequency from ```DLNAMediaAudio``` they are separate values and one can be used one place and another another place. We should see both values when debugging, if they are different that would be a bug.